crates/utils/Cargo.toml

```diff
@@ -25,12 +25,11 @@ derive_builder = "0.20.0"
 event-listener = "5.3.1"
 futures = "0.3.30"
 getset = "0.1"
+interval_map = { package = "rb-interval-map" }
```

Comment:
how about adding `version = "0.1.2"`? This line caused an error on rust-analyzer:
```
2024-08-26T00:21:28.930391Z ERROR FetchWorkspaceError
...
Caused by:
  dependency (interval_map) specified without providing a local path, Git repository, version, or workspace dependency to use
```

Reply:
adding `version = "0.1"` is ok, as this crate might require further modifications